jaeger-influxdb/internal/common.go

```diff
@@ -191,7 +193,40 @@ func recordToLog(record map[string]interface{}) (model.TraceID, model.SpanID, *m
 			} else {
 				log.Fields = append(log.Fields, model.String("message", vv))
 			}
-		case common.AttributeAttributes, semconv.AttributeServiceName:
+		case common.AttributeAttributes:
+			var m map[string]interface{}
+			if err := json.Unmarshal([]byte(v.(string)), &m); err != nil {
+				return model.TraceID{}, 0, nil, fmt.Errorf("")
```

Comment:
It's probably quite unlikely for this to get hit, but I wonder if it'd be good to note that invalid JSON has been passed into this function through the error so that anyone looking at the log isn't scratching their head?

Reply:
Thanks! That dangling error also pointed me to the fact that this case doesn't follow the same patterns as the sibling cases.

Fixed, and tested with the Jaeger demo.